jenkins/common/sonic-wpa-supplicant-build/Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+    agent { node { label 'sonic-slave-buster' } }
+
+    options {
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10'))
+    }
+
+    environment {
+        SONIC_TEAM_WEBHOOK = credentials('public-jenkins-builder')
+    }
+
+    triggers {
+        pollSCM('H/10 * * * *')
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                dir('sonic-wpa-supplicant') {
+                    checkout([$class: 'GitSCM',
+                          branches: [[name: 'refs/heads/master']],
+                          userRemoteConfigs: [[url: 'http://github.com/Azure/sonic-wpa-supplicant']]])
+                }
+                copyArtifacts(projectName: '../vs/buildimage-vs-all', filter: '**/*.deb', target: 'buildimage', flatten: false)
+            }
+        }
+
+        stage('Build') {
+            steps {
+                sh './scripts/common/sonic-wpa-supplicant/build.sh'
```

Comment:
The PATH is different with the following script.

Reply:
Fix it in this PR: https://github.com/Azure/sonic-build-tools/pull/195/
Please review it.